zim/plugins/toolbar.py

```diff
@@ -73,7 +75,7 @@ class ToolBarPlugin(PluginClass):
 
 	plugin_preferences = (
 		# key, type, label, default
-		('show_headerbar', 'bool', _('Show controls in the window decoration') + '\n' + _('This option requires restart of the application'), True),
+		('show_headerbar', 'bool', _('Show controls in the window decoration') + '\n' + _('This option requires restart of the application'), DEFAULT_DECOR),
```

Comment:
The manual states that the controls in the window decoration "is the default without the plugin", so this likely won't cut it.